core/blockchain/blockchain_gateway.py

```diff
@@ -1,164 +1,157 @@
-import asyncio
 import logging
-import ipfsapi
+from threading import Event, Timer
 import time
-from typing import Callable
+from typing import Callable, Tuple
+
+import ipfsapi
 
-from core.blockchain.blockchain_utils   import setter, getter, filter_diffs
-from core.blockchain.blockchain_utils   import get_global_state, ipfs_to_content
-from core.utils.event_types             import RawEventTypes
-from core.blockchain.tx_utils           import TxEnum
+from core.blockchain.blockchain_utils   import (filter_diffs, TxEnum,
+                                                get_global_state, ipfs_to_content)
+from core.utils.enums                   import RawEventTypes, MessageEventTypes
```

Comment:
not sure @neeleshdodda44

Reply:
@nzoghb can you clarify what you mean? Generally, we separate imported modules and packages with a line.

Reply:
I'm talking about the indents between `from core.utils.enums` and `import RawEventTypes, MessageEventTypes` for example

Reply:
Oh yeah so that applies for importing from modules. Not all of our files follow that convention currently, but we should clean up PR at some point to handle that.

Reply:
does it not apply to packages?